### PR TITLE
F/wep 121 re implement via normalizenode

### PR DESCRIPTION
**Pull request by faebebin**
This PR adds some validation and auto-fixes to the editor table element. Many more auto-fix procedures can be added on the table and other Elements in future PRs.

### CODE REVIEW

File: packages/editor/src/client/blocks/richTextBlock/editor/elements.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {BlockFormat} from './formats'
+import {Element as SlateElement} from 'slate'
+
+export function emptyTextParagraph(): SlateElement {
```
Comment (roosnic1) [inline packages/editor/src/client/blocks/richTextBlock/editor/elements.ts:4]: There is already a function called `createDefaultValue()` in the file `packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx`

File: packages/editor/src/client/tsconfig.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   // This file is only needed for VSCode, client will be built by webpack/babel.
   "extends": "../../../../tsconfig.base.json",
-  "include": ["./**/*"],
+  "include": ["./**/*", "../../__tests__/normalizing.test.ts"],
```
Comment (roosnic1) [inline packages/editor/src/client/tsconfig.json:4]: This shouldn't be needed.

File: packages/editor/src/client/blocks/richTextBlock/editor/normalizing.ts [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+export function withNormalizeNode<T extends ReactEditor>(editor: T): T {
+  const {normalizeNode} = editor
+
+  editor.normalizeNode = entry => {
+    const [node, path] = entry
+
+    if (SlateElement.isElement(node)) {
+      switch (node.type) {
+        // order of corrections is important !
```
Comment (roosnic1) [inline packages/editor/src/client/blocks/richTextBlock/editor/normalizing.ts:19]: Maybe improve this comment with a small but detailed explanation what exactly is happening here.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Looks very good. Just some minor improvements needed.

File: packages/editor/src/client/tsconfig.json (hunk 2 above)
Comment (faebebin) [reply packages/editor/src/client/tsconfig.json:4]: Oops, removed. thanks

File: packages/editor/src/client/blocks/richTextBlock/editor/elements.ts (hunk 1 above)
Comment (faebebin) [reply packages/editor/src/client/blocks/richTextBlock/editor/elements.ts:4]: Did it the other way round: `createDefaultValue() { return emptyTextParagraph ... `

**Review by roosnic1 — APPROVED** [review]
🎉

**Review by 0815pascal — APPROVED** [review]
Looks good 👍🏼

**Review by penk85 — APPROVED** [review]

**Pull request merged**